src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
There is a syntax error on this line: the string is not properly closed with a quotation mark. Make sure to close strings with matching quotation marks to prevent syntax errors.